pkg/convert/pulumiverse.go

```diff
@@ -56,6 +56,7 @@ var pulumiSupportedProviders = []string{
 	"kafka",
 	"keycloak",
 	"kong",
+	"kubernetes",
```

Comment:
This change seems unintentional here?  It should probably be in a different PR than the release PR.

Reply:
This PR is just on top of that one so I don't submit them in the wrong order, just filter by latest commit :)